flow-typed/npm/jest-when.js

```diff
@@ -0,0 +1,28 @@
+// NOTE(@somewhatabstract): Crafted from looking at the TypeScript types and
+// editing to match the Jest flow types for maintenance clarity.
```

Comment:
Created these types because I think they're useful and otherwise, we'd have to import `jest-when` as `any`-typed, which I want to clamp down on in this new repo.